subjects/java/checkpoints/good-by-mars/README.md

```diff
@@ -0,0 +1,36 @@
+## GoodbyMars
```

Comment:
I just realized that my comment had a typo: it should be `GoodbyeMars`
There are other places in the subject where `goodby` should be updated to `goodbye` or `Good by` to `Goodbye`

Reply:
Me too i didn't notice it, I'm fasting hhhhhh

Reply:
okay i will fix it thanks

Reply:
The name of the directory should change as well: `goodbye-mars` I guess would be the case :)